django_ipam/api/views.py

```diff
@@ -0,0 +1,16 @@
+import swapper
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+Subnet = swapper.load_model('django_ipam', 'Subnet')
+
+
+@api_view(['GET'])
+def iprequest(request):
```

Comment:
add a docstring to this function, it should show up in the browsable API, we can use that as a first basic version of API docs